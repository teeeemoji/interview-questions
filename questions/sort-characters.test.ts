import sortCharacters from './sort-characters'

test('Type', () => {
  expect(sortCharacters('Type')).toMatchSnapshot()
})

test('BabA', () => {
  expect(sortCharacters('BabA')).toMatchSnapshot()
})

test('By?e', () => {
  expect(sortCharacters('By?e')).toMatchSnapshot()
})

test('A Famous Saying: Much Ado About Nothing (2012/8).', () => {
  expect(sortCharacters('A Famous Saying: Much Ado About Nothing (2012/8).')).toMatchSnapshot()
})