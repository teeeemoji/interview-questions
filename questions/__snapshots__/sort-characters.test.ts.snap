// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A Famous Saying: Much Ado About Nothing (2012/8). 1`] = `"A aaAAbc dFgghh: iimM nNn oooos Sttuuuy (2012/8)."`;

exports[`BabA 1`] = `"aABb"`;

exports[`By?e 1`] = `"Be?y"`;

exports[`Type 1`] = `"epTy"`;
